media-gfx/drawpile/drawpile-2.0.10.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A collaborative drawing program"
+DESCRIPTION="Drawpile is a drawing program that lets you share \
+the canvas with other users in real time."
+HOMEPAGE="https://drawpile.net"
+COPYRIGHT="2006-2018 Calle Laakkonen"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/drawpile/Drawpile/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="0ff79e9f95ca02ff016eed3118a83ca62e80a1ca93fe11e3fb8ecd22c1fe37a9"
+SOURCE_DIR="Drawpile-$portVersion"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	drawpile$secondaryArchSuffix = $portVersion
+	app:Drawpile = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libkf5archive$secondaryArchSuffix
+	lib:libminiupnpc$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libsodium$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libarchive$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+	devel:libkf5archive$secondaryArchSuffix
+	devel:libminiupnpc$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libsodium$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. -DCMAKE_EXE_LINKER_FLAGS=-lnetwork
+	make $jobArgs
+}
+
+INSTALL()
+{
+	# TODO Add a rdef icon for Drawpile
+	mkdir -p $appsDir/Drawpile
+	cp bin/drawpile $appsDir/Drawpile/Drawpile
+	cp bin/drawpile-srv $appsDir/Drawpile/drawpile-srv
+	addAppDeskbarSymlink $appsDir/Drawpile/Drawpile
+}
+
+TEST()
+{
+	make test
+}
```

Comment:
Some editors don't put a LF (line-feed) at the end of the last line, but `vi(m)` does not like this very much, so if you could add one here, it would be cool :)

(`vi(m)` shows a `[Incomplete last line]` warning when we open a file that is missing the LF character at its last line.)

You can "fix" this "issue" with:
```bash
echo wq | ed drawpile-2.0.10.recipe
```

Thanks!